code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1428,6 +1428,17 @@
 	if(wear_id)
 		ACL |= wear_id.GetAccess()
 	return ACL
+	
+/mob/living/carbon/human/get_visible_id()
+	var/id = null
+	if(wear_id)
+		id = wear_id.GetID()
+	for(var/obj/item/I in held_items)
+		if(!id)
```

Comment:
This check should really go outside the `for` loop, and the loop should break when it finds an id